src/software/jetson_nano/display/screens/wheels_screen.py

```diff
@@ -0,0 +1,84 @@
+from screen import Screen
+
+ENABLE_INDEX = 0
+FL_INDEX = 1
+FR_INDEX = 2
+BL_INDEX = 3
+BR_INDEX = 4
+
+
+class WheelsScreen(Screen):
+    """
+    This screen is used to edit wheel speed settings
+    """
+
+    def __init__(self, lcd_display, redis_dict, screen_actions):
+        """
+        @param lcd_display, an instance of the LcdDisplay class
+        @param redis_dict, a dict of values from redis client to init variables on this screen
+        @param screen_actions, an instance of ScreenActions class
+        """
+        actions = [
```

Comment:
for all the actions, lets just add [m/s] units in the display string

Reply:
added